CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 <!-- Do not manually edit this file. Use the `changelogger` tool. -->
+November 21st, 2023
+===================
```

Comment:
we expect no changes here? We should probably add something like "no smithy-rs changes with this release" or something

Reply:
Added `Internal changes only with this release` in bab0341 and updated [the release notes](https://github.com/smithy-lang/smithy-rs/releases/tag/release-2023-11-21).